opendevin/README.md

```diff
@@ -3,11 +3,7 @@
 This directory contains the core components of OpenDevin.
 
 This diagram provides an overview of the roles of each component and how they communicate and collaborate.
-
-<div style="text-align: center;">
-  <img src="https://github.com/OpenDevin/OpenDevin/assets/16201837/97d747e3-29d8-4ccb-8d34-6ad1adb17f38" alt="OpenDevin System Architecture Diagram Jul 4 2024" />
-  <p><em>OpenDevin System Architecture Diagram (July 4, 2024)</em></p>
-</div>
+![OpenDevin System Architecture Diagram (July 4, 2024)](https://github.com/user-attachments/assets/542332d2-21c0-45dc-9ae3-a4eabc8d7ac3)
```

Comment:
I'm actually not sure if linking this `user-attachements` directly will work permanently or if it will disappear after a while? I think it might be better to link to an image included in the repo, like we do here for example: https://github.com/OpenDevin/OpenDevin/blob/437e0c76bff9057f16f8bb67b90a2fb329d611e7/README.md#L47

Reply:
Done!